### PR TITLE
Adds deploy github actions for prod builds

### PR DESCRIPTION
**Pull request by orta**
Separates out the metadata and build process for 'production builds' of the vscode extension

### CODE REVIEW

**Comment by orta** [comment]
re: #77

File: packages/svelte-vscode/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "url": "https://github.com/sveltejs/language-tools/issues"
     },
     "homepage": "https://github.com/sveltejs/language-tools#readme",
-    "displayName": "Svelte Beta",
+    "displayName": "Svelte for VS Code",
```
Comment (dummdidumm) [inline packages/svelte-vscode/package.json:26]: Bikeshedding: What do you think about naming it `Svelte (Official)`?

File: .github/workflows/Deploy.yml [hunk 2]
```diff
@@ -29,6 +29,9 @@ jobs:
       # Setup the environment
       - run: 'json -I -f packages/svelte-vscode/package.json -e "this.version=\`99.0.0\`"'
       - run: 'json -I -f packages/svelte-vscode/package.json -e "this.preview=true"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.name=\`svelte-vscode-nightly\`"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.icon=\`icons/logo-nightly.png\`"'
+      - run: 'json -I -f packages/svelte-vscode/package.json -e "this.displayName=\`Svelte Language Tools Nightly Builds\`"'
```
Comment (dummdidumm) [inline .github/workflows/Deploy.yml:34]: Bikeshedding#2: I think we should name the nightly almost the same as the stable one, just with a Nightly at the end. So for example `Svelte for VSCode (Nightly)` or `Svelte (Official, Nightly)`.

File: packages/svelte-vscode/package.json (hunk 1 above)
Comment (orta) [reply packages/svelte-vscode/package.json:26]: Not a fan personally, it feels a bit like calling yourself cool

Given the birtle's one will be deprecated it doesn't really need to stand out in that way

File: .github/workflows/Deploy.yml (hunk 2 above)
Comment (orta) [reply .github/workflows/Deploy.yml:34]: I'm not against it, but I choose this originally _because_ it's a bit more technically intimidating - people shouldn't really need to use the daily build very often and giving it the more unfriendly name was to discourage folks. We do the same for [TS' nightly builds](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next) in vscode

File: packages/svelte-vscode/package.json (hunk 1 above)
Comment (dummdidumm) [reply packages/svelte-vscode/package.json:26]: Fair enough. After Birtle's is deprecated/renamed/removed, we could think about renaming it to just "Svelte"?

File: .github/workflows/Deploy.yml (hunk 2 above)
Comment (dummdidumm) [reply .github/workflows/Deploy.yml:34]: I get the intimidating part, but I still feel like people should at least be able to discover it easily. What about `<Original name> Unstable Nightly Builds` instead? The `Unstable` will trigger most people to not use it in their day-to-day-work.

Comment (orta) [reply .github/workflows/Deploy.yml:34]: Works for me

File: packages/svelte-vscode/package.json (hunk 1 above)
Comment (orta) [reply packages/svelte-vscode/package.json:26]: Legit

**Comment by orta** [comment]
I've shipped some changes for the old extension: https://github.com/orta/svelte-vscode-unwrittenfun/pull/1

**Pull request merged**